test/unit/headset/headset.test.ts

```diff
@@ -496,7 +496,7 @@ describe('HeadsetProxyService', () => {
     }
 
     beforeEach(() => {
-      jest.useFakeTimers();
+      jest.useFakeTimers('legacy');
```

Comment:
This was mentioned in the post about jest v26: https://jest-archive-august-2023.netlify.app/blog/2020/05/05/jest-26/#new-fake-timers

I did a little digging, but it's not obvious to me why the new fake timers are a problem for our tests, so I'll make a ticket for us to dig into more.

Reply:
What issues were you seeing? I'm pretty sure we don't do anything crazy with timers.

Reply:
It looks like tests were taking too long with the new fake timers (or, now that I think about it, perhaps the tests weren't actually using the fake timers, which throws off the behavior?). For example, `thrown: "Exceeded timeout of 5000 ms for a test.` But using the legacy timers works just fine. I created https://inindca.atlassian.net/jira/software/c/projects/STREAM/issues/STREAM-15 to dig into this more. I might give it one more shot today, though.

Reply:
I did some more digging and found this solution which seems to work: https://gist.github.com/apieceofbart/e6dea8d884d29cf88cdb54ef14ddbcc4?permalink_comment_id=4248861#gistcomment-4248861 (It looks like Jest will fake `process.nextTick`, so if we capture that before it gets faked, we can still use it.)

I also saw [another comment](https://gist.github.com/apieceofbart/e6dea8d884d29cf88cdb54ef14ddbcc4?permalink_comment_id=4760758#gistcomment-4760758) that says Jest v29 will let us tell Jest to not fake that: `jest.useFakeTimers({ doNotFake: ['nextTick'] })`. Since @zservies is working on updating dependencies, I'm leaning towards just letting him make that change.

Reply:
Small update. Looks like Jest v28 also lets us define what not to fake: https://jest-archive-august-2023.netlify.app/docs/28.x/jest-object#jestusefaketimersfaketimersconfig